extensions/widgets/layout/src/components/layout-widget.tsx

```diff
@@ -246,7 +248,7 @@ const Layout: React.FC<unknown> = () => {
               />
             </Stack>
 
-            <Stack customStyle="fixed bottom-2 mx-2 lg:(w-[21rem])">
+            <Stack customStyle="fixed bottom-2 ml-5 mr-2 lg:(w-[21rem])">
```

Comment:
The widget container has position fixed thats required to avoid a flicker on widget area while scrolling. In connection the margin left is needed :) I have refined this a bit for it to look clean.